api.js

```diff
@@ -81,10 +81,21 @@ Api.prototype._handleOutput = function (channel, data) {
 	this.emit(channel, data);
 };
 
+function normalizeError(err) {
+	if (!err || typeof err !== 'object') {
```

Comment:
Would maybe be clearer to use something like: https://github.com/sindresorhus/is-obj